src/PrintFeature.h

```diff
@@ -1,12 +1,14 @@
 #ifndef PRINT_FEATURE
 #define PRINT_FEATURE
 
+#include <cstdint>
+
 namespace cura
 {
 
-enum class PrintFeatureType
+enum class PrintFeatureType: std::uint8_t
 {
-    NoneType, // unused, but libArcus depends on it
+    NoneType, // used to mark unspecified jumps in polygons. libArcus depends on it
```

Comment:
I mean there is a movement in the path that has not been specified by added data. This could for instance be if you were to send the line data before the export step where you only have isolated infill lines. If they were added to a path segment the NoneType would indicate this jump.